Description: Upstream changes introduced in version 1.3.2-0oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ros-fuerte-octomap (1.3.2-0oneiric) oneiric; urgency=high
 .
   * autogenerated
 .
 The person named in the Author field signed this changelog entry.
Author: Armin Hornung <hornunga@informatik.uni-freiburg.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ ros-fuerte-octomap-1.3.2/stack.yaml
@@ -0,0 +1,13 @@
+Catkin-ProjectName: octomap
+Version: 1.3.2
+
+Depends: libstdc++6
+
+Description: The OctoMap library implements a 3D occupancy grid mapping approach, providing data structures and mapping algorithms in C++. The map implementation is based on an octree.
+
+Author: Kai M. Wurm and Armin Hornung
+Maintainer: Armin Hornung <hornunga@informatik.uni-freiburg.de>
+Homepage: http://octomap.sourceforge.net/
+
+Catkin-CopyrightType:  willowgarage
+Catkin-DebRulesType: cmake
--- /dev/null
+++ ros-fuerte-octomap-1.3.2/catkin.conf
@@ -0,0 +1,3 @@
+[catkin]
+    upstream =  http://sourceforge.net/projects/octomap/files/octomap-1.3.1.tar.gz/download
+    upstreamtype = manual
